src/odemis/gui/cont/tabs/cryo_chamber_tab.py

```diff
@@ -910,6 +924,37 @@ def _sample_heater_to_va(self):
     def Show(self, show=True):
         Tab.Show(self, show=show)
 
+        # the first time the tab is shown, ask the user to create or load a project
+        if self.IsShown() and not self._is_initial_project_ready:
+
+            # stay in the loop until the user successfully creates or loads a project
+            while True:
+
+                # Ask the user to create or load a project
+                box = wx.MessageDialog(
```

Comment:
Maybe you can handle this dialog and the subsequent dialogues (steps -> pages (buttons) in Wizard) using https://docs.wxpython.org/wx.adv.Wizard.html

Just a suggestion!

Reply:
Interesting, didn't know about this! Probably not for this PR, but might be worth re-doing the create/load project workflow with this